src/graphene.cpp

```diff
@@ -792,6 +792,91 @@ double CGrapheneBlockData::average(std::map<int64_t, uint64_t> &map)
     return (double)accum / map.size();
 }
 
+double CGrapheneBlockData::computTotalBandwidthSavingsInternal() EXCLUSIVE_LOCKS_REQUIRED(cs_graphenestats)
```

Comment:
Typo in the function name? 'comput'

Reply:
Will fix.  I also noticed some spelling/typos in some of the comments as well.

Reply:
Fixed.